pump/templates/main/response.html

```diff
@@ -383,7 +386,7 @@ <h3>Email me my results and additional resources:</h3>
 <input type="hidden" name="response" value="{{object.id}}" />
 <div class="form-group">
 <label for="email">Email</label>
-<input type="text" name="email" id="email" class="form-control" placeholder="enter your email address" />
+<input type="email" name="email" id="email" class="form-control" placeholder="enter your email address" />
```

Comment:
The HTML5 email validation is quite good. You can also add the attribute `required` to ensure the user specifies a value before hitting submit.